### DISCUSSION TITLE
Something about Favorites

### DISCUSSION THREAD

**Discussion by xishizhe** [community/community#3964] [Mobile] [answered]
Hello, I hope the way of adding Repositories to Favorites can be increased on the phone.

  At present, I can only add them through the Edit button on the right side of the Favorites option in the Home interface. This leads to that
every time you want to add a Repository that you are watching to Favorites, you have to exit the Repository and go back to the Home interface, click the Edit button on the right side of Favorites, and then search again to add it.

  Although you can also give star to a Repository to preserve it, you can only see it in the profile interface. In contrast, the Favorites in the Home interface is obviously more convenient, but the way to add it is a little more complicated.

  I hope some buttons or options can be added to the Repositories details interface that can directly add Repositories to Favorites.
  Thanks!

![1](https://user-images.githubusercontent.com/55939287/118360350-bf6b7580-b5b9-11eb-8253-bab2582791e9.jpg)

![2](https://user-images.githubusercontent.com/55939287/118360352-c1353900-b5b9-11eb-825d-f1c0c7c24d9a.jpg)

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for this request! We are tracking this internally and I just added your note to our internal issue. Please keep the feature requests and/or bug reports coming! We really appreciate it.

**Reply by xishizhe** [reply]
You're welcome.
Hope to see the changes of it, thanks for your hard work.
